widget/embedded/src/components/HeaderButtons/UnreadNotificationsBadge.tsx

```diff
@@ -0,0 +1,20 @@
+import { Typography } from '@rango-dev/ui';
+import React from 'react';
+
+import { useNotificationStore } from '../../store/notification';
+
+import { NotificationsBadgeContainer } from './HeaderButtons.styles';
+
+export function UnreadNotificationsBadge() {
+  const notificationsCount = useNotificationStore.use
+    .notifications()
+    .filter((item) => !item.read).length;
```

Comment:
could we move this logic to zustand state? adding a `unread_notifications` method for example.

Reply:
Done